### ISSUE TITLE
Null reference exception (fix proposed)

### ISSUE THREAD

**Issue by AdamGolovenko** [gave92/fbchat-sharp#20]
I tested library, found Null reference exception in company chats (quick replies) - see file Message.cs, line 281:
```
var quick_replies = JToken.Parse(data.get("platform_xmd_encoded")?.Value<string>()).get("quick_replies");
if (quick_replies.Type == JTokenType.Array)
    rtn.quick_replies = quick_replies.Select((q) => FB_QuickReply.graphql_to_quick_reply(q)).ToList();
else
    rtn.quick_replies = new List<FB_QuickReply>() { FB_QuickReply.graphql_to_quick_reply(quick_replies) };
```
In two company chats I received quick_replies  == null.
Propose small fix:
```
                var quick_replies = JToken.Parse(data.get("platform_xmd_encoded")?.Value<string>()).get("quick_replies");
                if (quick_replies != null)
                {
                    if (quick_replies.Type == JTokenType.Array)
                        rtn.quick_replies = quick_replies.Select((q) => FB_QuickReply.graphql_to_quick_reply(q)).ToList();
                    else
                        rtn.quick_replies = new List<FB_QuickReply>() { FB_QuickReply.graphql_to_quick_reply(quick_replies) };
                }
```

**Comment by AdamGolovenko** [comment]
* company chats = business page chats

**Comment by gave92** [comment]
Thanks! Fixed in 16bef50cbe682af217a30368d4901cf5f363d523
